internal/oci/runtime_oci.go

```diff
@@ -45,6 +47,15 @@ const (
 	rootFlag = "--root"
 )
 
+var (
+	// Configuration for the stop loop exponential backoff manager.
```

Comment:
**To the reviewer:**

Hopefully, these values are somewhat acceptable. We can always raise the maximum and reset backoff limits as needed to make the loop stagger slightly more.